dash/testing/browser.py

```diff
@@ -159,7 +159,10 @@ def percy_snapshot(
         """
         if widths is None:
             widths = [1280]
-        snapshot_name = f"{name} - py{sys.version_info.major}.{sys.version_info.minor}"
+
+        # py3.9 hardcoded here to keep snapshot names the same accorss
+        # future python upgrades
+        snapshot_name = f"{name} - py3.9"
```

Comment:
Ah that's a good strategy - this means we know what changes if any the rest of the changes in this PR cause in the percy snapshots. After merging we can make another PR to remove the version number entirely, which will cause all the snapshots to look new but we can blindly approve.